README.md

```diff
@@ -1,5 +1,29 @@
 # studio extension
 
-Run in development mode:
+## Installing a development version
 
-`yarn watch --extension-folder ~/git/projectatomic/packages/backend`
+You can install this extension from Podman Desktop UI > ⚙ Settings > Extensions > Install a new extension from OCI Image.
+
+The name of the image to use is `ghcr.io/projectatomic/studio-extension:latest`.
+
+You can get earlier tags for the image at https://github.com/projectatomic/studio-extension/pkgs/container/studio-extension.
+
+These images contain development versions of the extension. There is no stable release yet.
+
+## Running in development mode
+
+From the Podman Desktop sources folder:
+
+```
+$ yarn watch --extension-folder path-to-extension-sources-folder/packages/backend
+```
+
+## Providing a custom catalog
+
+The extension provides a default catalog, but you can build your own catalog by creating a file `$HOME/podman-desktop/ai-studio/catalog.json`.
```

Comment:
maybe we can extend by showing where the file is in the source code  so the user knows how to conform to the format and mentioning to do a PR.

Reply:
it's not related to the readme but probably folder should not contain the name podman-desktop as Podman Desktop is not owning it, it's an extension choice.

Reply:
--> `$HOME/ai-studio`